components/net/http_loader.rs

```diff
@@ -1248,11 +1248,9 @@ fn cors_preflight_fetch(request: Rc<Request>,
 
     // Step 3, 4
     let mut value = request.headers.borrow().iter()
-                                            .filter_map(|ref view| if is_simple_header(view) {
-                                                None
-                                            } else {
-                                                Some(UniCase(view.name().to_owned()))
-                                            }).collect::<Vec<UniCase<String>>>();
+                                            .filter(is_simple_header)
```

Comment:
Previously, this was filtering out if `is_simple_header(..) == true`. Now it's doing the opposite?